changelogs/current.yaml

```diff
@@ -95,5 +95,8 @@ new_features:
 - area: mobile
   change: |
     started merging the Envoy mobile library into the main Envoy repo.
+- area: redis
+  change: |
+    added support for DNS resolving endponts returned by MOVED/ASK responses.
```

Comment:
Can you ref link to the new field and also the arch docs that I mention adding above?